packages/ui/src/services/cache/plugin/index.ts

```diff
@@ -0,0 +1,25 @@
+import type { App } from 'vue'
+import { AppCache } from '../types'
+import { reactive } from 'vue'
+import { defineVuesticPlugin } from '../../../vuestic-plugin/utils'
+
+const createAppCachePlugin = (app: App) => {
+  const cache: AppCache = ({
+    colorContrast: {},
+  })
+
+  return cache
+}
```

Comment:
The `app` parameter is not used. If there will be just creating an object, then I'll suggest this:
```suggestion
const createAppCachePlugin = (): AppCache => ({ colorContrast: {} })
```